libs/agno/agno/storage/dynamodb.py

```diff
@@ -30,6 +43,7 @@ def __init__(
         endpoint_url: Optional[str] = None,
         create_table_if_not_exists: bool = True,
         mode: Optional[Literal["agent", "team", "workflow", "workflow_v2"]] = "agent",
+        provisioned_throughput: Optional[DynamoDbProvisionedThroughput] = None,
```

Comment:
No need for a class I think, we can just have `read_capacity_units` and `write_capacity_units` directly here and process it for Dynamo further down